src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -445,20 +413,13 @@ describe('<ComboBox/>', () => {
           onSelect={spy}
         />,
       );
-
-      async () => {
-        comboBox.find(TextField).simulate('click');
-        await listenerMap.keyup({keyCode: Key.DownArrow});
-      };
-
-      async () => {
-        listenerMap.keyup({keyCode: Key.Enter});
-        await expect(spy).toHaveBeenCalledTimes(1);
-        expect(comboBox.prop('selected')[0]).toBe('cheese_pizza');
-      };
+      comboBox.find(TextField).simulate('click');
+      dispatchKeyup(Key.DownArrow);
+      dispatchKeyup(Key.Enter);
+      expect(spy).toHaveBeenCalledWith(['cheese_pizza']);
     });
 
-    it('activates the popover when the tab key is pressed', () => {
```

Comment:
From what I could tell we don't listen for a `tab` keypress, we're trying to test if focus activates the popover